quilt/tools/command.py

```diff
@@ -243,6 +244,10 @@ def push(session, package):
         ))
     )
 
+    if not response.ok:
```

Comment:
I don't think this can happen - it should already be handled by `_handle_response`.